src/sampled_exp_val.rs

```diff
@@ -0,0 +1,93 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2022
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use num_complex::Complex64;
+
+use hashbrown::HashMap;
+use numpy::PyReadonlyArray1;
+use pyo3::prelude::*;
+use pyo3::wrap_pyfunction;
+
+use crate::pauli_exp_val::fast_sum;
+
+const fn generate_oper_table() -> [[f64; 2]; 91] {
+    let mut table = [[0.; 2]; 91];
```

Comment:
This is a good call, I'll do this locally though because I should update all the usage of `91` and also I think cargo fmt will complain with it being lower case

Reply:
Oh yeah. I tested it locally but forgot to run the formatting tool.